packages/user-management-widget/src/lib/widget/mockTemplates/createUserTemplate.ts

```diff
@@ -9,6 +9,8 @@ export default `
   st-vertical-padding="0rem"
   st-border-radius="sm"
 >
+  <descope-combo-box bordered="true" data-id="roles-multiselect"></descope-combo-box>
```

Comment:
Replace with MultiSelect component

Reply:
And move to place

Reply:
This should be added to the page schema after we are adding the multi-select to the flow components